collector.go

```diff
@@ -67,12 +122,33 @@ type ChunkedCollector struct {
 	started, stopped bool
 	stopChan         chan struct{}
 
+	queueSizeBytes  uint64
 	pendingBySpanID map[SpanID]Annotations
 
 	// mu protects pendingBySpanID, lastErr, started, stopped, and stopChan.
 	mu sync.Mutex
 }
 
+// NewChunkedCollector is shorthand for:
+//
+// 	c := &ChunkedCollector{
+// 		Collector:    c,
+// 		MinInterval:  500 * time.Millisecond,
+// 		FlushTimeout: 50 * time.Millisecond,
+// 		MaxQueueSize: 32 * 1024 * 1024, // 32MB
+// 		Log:          log.New(os.Stderr, "appdash: ", log.LstdFlags),
+// 	}
```

Comment:
Not sure if it's great to just copy paste the code here. I understand you're doing it to document what this func does exactly.

However, what about phrasing it as "NewChunkedCollector creates a ChunkedCollector with optimal default values." Those who really want to find out what the default values are can just look it up in the code.

Less maintenance and less chance of...

![image](https://cloud.githubusercontent.com/assets/1924134/13589678/86632a50-e48e-11e5-8462-7a441463a3d7.png)